library/src/main/java/com/readystatesoftware/chuck/internal/data/room/ChuckerDatabase.kt

```diff
@@ -0,0 +1,26 @@
+package com.readystatesoftware.chuck.internal.data.room
+
+import android.arch.persistence.room.Database
+import android.arch.persistence.room.Room
+import android.arch.persistence.room.RoomDatabase
+import android.content.Context
+import com.readystatesoftware.chuck.internal.data.entity.HttpTransaction
+import com.readystatesoftware.chuck.internal.data.entity.RecordedThrowable
+
+@Database(entities = [RecordedThrowable::class, HttpTransaction::class], version = 1, exportSchema = false)
```

Comment:
The database version is going back to `1`? What about the previous data?

Reply:
So for this, I won't spend much time doing migrations as this is going to be a breaking change.
We also need to change the package name so I don't expect users complaining because their transactions/throwables are not stored.

What do you think?

Reply:
Ok for me. Actually Chucker is a dev tool, nothing of this should go to prod so we don't really care of migration.
BTW if I understand well the previous data is still there (in antoher DB), can we deleted it ? If not... don't worry, this is just a nice to have :-)

Reply:
> BTW if I understand well the previous data is still there (in antoher DB), can we deleted it ? If not... don't worry, this is just a nice to have :-)

So yeah, I pushed a commit to delete the old db: e04c085
I'm just afraid this will make Chucker incompatible with Chuck if used simultaneously. I don't really know if this is a real use-case. But probably something worth noting.